lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -166,7 +166,7 @@ Var Js::InterpreterStackFrame::INTERPRETERLOOPNAME()
     }
 
     Assert(this->returnAddress != nullptr);
-    AssertMsg(m_arguments == NULL || Js::ArgumentsObject::Is(m_arguments), "Bad arguments!");
```

Comment:
I intended to do that. But looks like FunctionBody has no api for hasArgumentObject. I did not add a bool on FunctionBody as it is expensive.